sormas-app/app/src/main/java/de/symeda/sormas/app/person/edit/PersonEditFragment.java

```diff
@@ -463,6 +466,9 @@ public void onLayoutBinding(FragmentPersonEditLayoutBinding contentBinding) {
 	@Override
 	public void onAfterLayoutBinding(final FragmentPersonEditLayoutBinding contentBinding) {
 		PersonEditFragment.setUpLayoutBinding(this, record, contentBinding, rootData);
+		if (!ConfigProvider.isConfiguredServer(CountryHelper.COUNTRY_CODE_GERMANY)) {
```

Comment:
Hiding the field can be achieved by adding the `@HideForCountriesExcept` annotation on the `armedForcesRelationType` field of `PersonDto`

Reply:
I have added the @HideForCountriesExcept notation in the PersonDto. It did not seem to work for the App in my case.

Reply:
seems like Person fragments don't call `setFieldVisibilitiesAndAccesses`.

See `EventReadFragment::onAfterLayoutBinding` for a simple example

Reply:
Maybe the `setFieldVisibilitiesAndAccesses` is not used here for a reason. It seems to break the enums (they don't load when the setFieldVisibilitiesAndAccesses` is used in the personEditFragment). Do you happen to have an explanation for this? Otherwise I would like to leave this as is, because I don't have time to look into it and my implementation also works.

Reply:
Ok, I'll find a solution when doing the salutation on https://github.com/hzi-braunschweig/SORMAS-Project/pull/3604